integration-tests/platform.spec.js

```diff
@@ -128,12 +128,11 @@ describe('platform end-to-end', function () {
                 expect(err).toBeUndefined();
             }).fin(done);
     });
-
-    it('Test 002 : should install plugins correctly while adding platform', function (done) {
-
-        cordova.plugin('add', path.join(pluginsDir, 'test'))
+    // was using old fetch code/gitclone... test might not be valid anymore.
+    xit('Test 002 : should install plugins correctly while adding platform', function (done) {
```

Comment:
Did you mean to have this `xit`

Reply:
i think it was was using old fetch code/gitclone... wasn't sure if the test was still valid. I took out the "x" so we can see if it passes/fails.